src/constants.ts

```diff
@@ -61,66 +61,43 @@ export const PRESENTER_ALREADY_ADDED_PAGE = "presenter-already-added";
 export const MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_EMAIL_RESENT_URL = `${MANAGE_AUTHORISED_PEOPLE_URL}${AUTHORISATION_EMAIL_RESENT_URL}`;
-export const YOUR_COMPANIES_CONFIRMATION_EMAIL_RESENT_URL = `${LANDING_URL}${MANAGE_AUTHORISED_PEOPLE_URL}${AUTHORISATION_EMAIL_RESENT_URL}`;
-export const ADD_PRESENTER_URL = `${SEPARATOR}${ADD_PRESENTER_PAGE}${SEPARATOR}:${COMPANY_NUMBER}`;
-export const YOUR_COMPANIES_ADD_PRESENTER_URL = LANDING_URL + ADD_PRESENTER_URL;
-export const CHECK_PRESENTER_URL = `${SEPARATOR}${CHECK_PRESENTER_PAGE}${SEPARATOR}:${COMPANY_NUMBER}`;
-export const YOUR_COMPANIES_CHECK_PRESENTER_URL = LANDING_URL + CHECK_PRESENTER_URL;
-export const COMPANY_INVITATIONS_URL = SEPARATOR + COMPANY_INVITATIONS_PAGE;
-export const YOUR_COMPANIES_COMPANY_INVITATIONS_URL = `${LANDING_URL}${COMPANY_INVITATIONS_URL}`;
+export const ADD_PRESENTER_URL = `/${ADD_PRESENTER_PAGE}/:${COMPANY_NUMBER}`;
+export const CHECK_PRESENTER_URL = `/${CHECK_PRESENTER_PAGE}/:${COMPANY_NUMBER}`;
+export const COMPANY_INVITATIONS_URL = `/${COMPANY_INVITATIONS_PAGE}`;
 export const COMPANY_INVITATIONS_ACCEPT_URL = `/${COMPANY_INVITATIONS_ACCEPT_PAGE}/:${ASSOCIATIONS_ID}`;
-export const YOUR_COMPANIES_COMPANY_INVITATIONS_ACCEPT_URL = LANDING_URL + COMPANY_INVITATIONS_ACCEPT_URL;
 export const COMPANY_INVITATIONS_DECLINE_URL = `/${COMPANY_INVITATIONS_DECLINE_PAGE}/:${ASSOCIATIONS_ID}`;
-export const YOUR_COMPANIES_COMPANY_INVITATIONS_DECLINE_URL = LANDING_URL + COMPANY_INVITATIONS_DECLINE_URL;
 export const CONFIRMATION_PERSON_REMOVED_URL = "/confirmation-person-removed";
-export const CONFIRMATION_PERSON_ADDED = "/confirmation-person-added";
 export const MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_PERSON_REMOVED_URL = `${MANAGE_AUTHORISED_PEOPLE_URL}${CONFIRMATION_PERSON_REMOVED_URL}`;
-export const YOUR_COMPANIES_MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_PERSON_REMOVED_URL = `${LANDING_URL}${MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_PERSON_REMOVED_URL}`;
 export const REMOVE_COMPANY_URL = `/${REMOVE_COMPANY_PAGE}/:${COMPANY_NUMBER}`;
-export const YOUR_COMPANIES_REMOVE_COMPANY_URL = `${LANDING_URL}${REMOVE_COMPANY_URL}`;
-export const REMOVED_THEMSELVES_URL = SEPARATOR + REMOVED_THEMSELVES;
-export const YOUR_COMPANIES_REMOVED_THEMSELVES_URL = LANDING_URL + REMOVED_THEMSELVES_URL;
+export const REMOVED_THEMSELVES_URL = `/${REMOVED_THEMSELVES}`;
 export const SOMETHING_WENT_WRONG_URL = "/something-went-wrong";
-export const YOUR_COMPANIES_SOMETHING_WENT_WRONG_URL = LANDING_URL + SOMETHING_WENT_WRONG_URL;
-
 export const REMOVE_ASSOCIATION_URL = `/:${COMPANY_NUMBER}/remove-association`;
-export const YOUR_COMPANIES_REMOVE_ASSOCIATION = LANDING_URL + REMOVE_ASSOCIATION_URL;
+export const HEALTHCHECK_URL = "/healthcheck";
 
-export const HEALTHCHECK = "/healthcheck";
+// External URLs
+export const CHANGE_COMPANY_AUTH_CODE_URL = "https://www.gov.uk/guidance/company-authentication-codes-for-online-filing#change-or-cancel-your-code";
```

Comment:
Sure this be in a translation file? or provide the welsh version somehow depending on the language? https://www.gov.uk/guidance/company-authentication-codes-for-online-filing.cy

Maybe we can raise a ticket for this if it's difficult to handle.

Reply:
I pulled this link to constants, as originally it was hard coded in the handler. I guess, this could be tackled in a ticket of it's own.